persistence/storage.go

```diff
@@ -1,7 +1,12 @@
 package persistence
 
+// JSONValue holds a json representation of a struct
+type JSONValue []byte
+
 // Storage backend to be used by the persistence layer
 type Storage interface {
-	Save(key int64, value []byte) (int64, error)
-	Get(key int64) ([]byte, error)
+	Save(key int64, value JSONValue) (int64, error)
```

Comment:
What is the meaning of the int64 return value? document it

Reply:
I'll remove it